nss-day-cohort-45/Hello-World-The-Animation-Durations
# Gt features

## Changes Made:

1. add basic info to Thailand page
2. add bottom border to cities, landmarks, and citizens classes
3. reformatted image size

## How to Test:

1. Fetch gt-features branch
2. load site via 'serve'
3. observe changes to site layout
